backend/de.metas.business/src/main/java/de/metas/bpartner/composite/repository/BPartnerCompositeSaver.java

```diff
@@ -159,9 +156,13 @@ private void saveBPartner(@NonNull final BPartner bpartner, @Nullable final OrgI
 
 		bpartnerRecord.setIsVendor(bpartner.isVendor());
 		bpartnerRecord.setIsCustomer(bpartner.isCustomer());
-		
+		if (!Check.isEmpty(bpartner.getInvoiceRule()))
```

Comment:
```suggestion
		if (bpartner.getInvoiceRule() != null)
```

i know we do have that method, but as the Invoicerule can be just null or nut-null (unlike a String or List), i think the code is clearer like this